dmutils/s3.py

```diff
@@ -96,5 +112,12 @@ def _get_mimetype(self, filename):
         return mimetype
 
 
+def get_file_size(file_contents):
+    size = len(file_contents.read(FILE_SIZE_LIMIT))
```

Comment:
It looks like this will only read a maximum of `FILE_SIZE_LIMIT` bytes?  In which case when testing the file size for a large file it will only ever be `FILE_SIZE_LIMIT`... the `file_is_less_than_5mb`  method will work OK because the test is for less than, but it seems odd, or at least wrongly named - could it be `get_file_size_up_to_maximum` to be clearer about what it does?